resources/frontend-development.md

```diff
@@ -0,0 +1,8 @@
+# Fontend Development
+## CSS
+
+Here is a list of useful websites for learning and mastering CSS:
+
+### Online Courses and Tutorials
+1. [Codecademy](https://www.codecademy.com/learn/learn-css)
+2. [freeCodeCamp](https://www.freecodecamp.org/learn/)
```

Comment:
```suggestion
- [Codecademy](https://www.codecademy.com/learn/learn-css)
- [freeCodeCamp](https://www.freecodecamp.org/learn/)
```

Please remove the `h3` and let's use `ul` instead of `ol`.